---
title: "' GoTo <labelname>' n’est pas valide, car '<labelname>' est à l’intérieur d’un 'Try', 'Catch' ou 'Finally' qui ne contient pas cette instruction"
ms.date: 07/20/2015
f1_keywords:
- bc30754
- vbc30754
helpviewer_keywords:
- BC30754
ms.assetid: 2eefc7fb-fdf0-41e9-bf60-c3bc93580e14
ms.openlocfilehash: 3b4d8c561622ebe310832b83a458162bab15a9d8
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: HT
ms.contentlocale: fr-FR
ms.lasthandoff: 04/23/2019
ms.locfileid: "62001882"
---
# <a name="goto-labelname-is-not-valid-because-labelname-is-inside-a-try-catch-or-finally-statement-that-does-not-contain-this-statement"></a>' GoTo \<nom_étiquette >' n’est pas valide, car '\<nom_étiquette >' est à l’intérieur d’un 'Try', 'Catch' ou 'Finally' qui ne contient pas cette instruction
Vous ne pouvez pas créer de branche dans un bloc `Try...Catch...Finally` .  
  
 **ID d’erreur :** BC30754  
  
## <a name="to-correct-this-error"></a>Pour corriger cette erreur  
  
- Restructurez votre code pour que l’étiquette précède le bloc `Try...Catch...Finally` .  
  
## <a name="see-also"></a>Voir aussi

- [Try...Catch...Finally (instruction)](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
